lms/djangoapps/instructor/enrollment.py

```diff
@@ -287,6 +297,43 @@ def _reset_module_attempts(studentmodule):
     studentmodule.save()
 
 
+def _fire_score_changed_for_block(course_id, student, block, module_state_key):
+    """
+    Fires a SCORE_CHANGED event for the given module. The earned points are
+    always zero. We must retrieve the possible points from the XModule, as
+    noted below.
+    """
+    if block and block.has_score:
+        cache = FieldDataCache.cache_for_descriptor_descendents(
+            course_id=course_id,
+            user=student,
+            descriptor=block,
+            depth=0
+        )
+        # For implementation reasons, we need to pull the max_score from the XModule, even though the data is not user-
+        # specific.  Here we bind the data to the current user.
+        request = crum.get_current_request()
+        module = get_module_for_descriptor(
+            user=student,
+            request=request,
+            descriptor=block,
+            field_data_cache=cache,
+            course_key=course_id
+        )
+        points_earned, points_possible = weighted_score(0, module.max_score(), getattr(module, 'weight', None))
+    else:
+        points_earned = 0
```

Comment:
Nit: for readability (to be consistent with the previous line), change this to:
`points_earned, points_possible = 0, 0`